src/lib/validation/utils/helpers.js

```diff
@@ -2,7 +2,7 @@ import getScrollTop from '../../../utils/getScrollTop'
 import prefixClass from '../../../utils/prefixClass'
 
 function getAllInputsArray (form) {
-  return Array.from(form.querySelectorAll('[pattern], [required], input[type="email"], .tiptap__input--hidden.is-required, .multiselect.is-required, fieldset.is-required, [minlength], [maxlength], [data-dp-validate-maxlength], [data-dp-validate-should-equal]'))
+  return Array.from(form.querySelectorAll('[pattern], [required], input[type="email"], input[name="r_startdate"], input[name="r_enddate"], .tiptap__input--hidden.is-required, .multiselect.is-required, fieldset.is-required, [minlength], [maxlength], [data-dp-validate-maxlength], [data-dp-validate-should-equal]'))
```

Comment:
I'd opt against hardcoding business logic from demosplan-core here. Instead, we could always validate date inputs with ```[data-ad-id]``` (which is produced by the datepicker lib).

Reply:
Hmm, if this is a specific Datepicker attribute, then we can do it. Thanks